v2/federation/fetch_test.go

```diff
@@ -0,0 +1,112 @@
+package federation_test
+
+import (
+	"context"
+	"net"
+	"testing"
+
+	"github.com/spiffe/go-spiffe/v2/bundle/spiffebundle"
+	"github.com/spiffe/go-spiffe/v2/federation"
+	"github.com/spiffe/go-spiffe/v2/internal/test"
+	"github.com/spiffe/go-spiffe/v2/internal/test/fake"
+	"github.com/spiffe/go-spiffe/v2/spiffeid"
+	"github.com/spiffe/go-spiffe/v2/svid/x509svid"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestFetchBundle_WebPKIAuth(t *testing.T) {
+	td := spiffeid.RequireTrustDomainFromString("domain.test")
```

Comment:
How would you feel about making this a package level variable? From what I can tell it is used in every test.

Reply:
I feel I must do it! Thanks!